docs/self-managed/identity/guides/configure-external-identity-provider.md

```diff
@@ -0,0 +1,11 @@
+---
+id: configure-external-identity-provider
+title: "Guide: Configuring an external identity provider"
+sidebar_label: "Configuring an external identity provider"
+---
+
+:::note
+Currently the Identity UI does not offer support for configuring external identity providers. To configure an external
+identity provider we recommend referencing the Keycloak documentation for 
+[adding a provider](https://www.keycloak.org/docs/16.1/server_admin/index.html#adding-a-provider)
+:::
```

Comment:
Question about this - I like that we have an answer to the question "can I configure an external identity provider?" but I only want to maintain a separate page if at some point in the future we'll plan to have steps to do this. Will we fully support this in the future? (We can also discuss this internally if it's a sensitive topic.)

Reply:
That is correct @akeller - we took the decision for now to direct users to some form of documentation however in future iterations the idea is that this will be supported within Identity and therefore will have its own set of steps (similar to the other pages)

Reply:
Gotcha, I'm good with this approach then!